code/game/objects/items/plushes.dm

```diff
@@ -827,3 +827,47 @@
 	icon_state = "unicorn"
 	attack_verb_continuous = list("whinnies", "gallops", "prances", "magicks")
 	attack_verb_simple = list("whinny", "gallop", "prance", "magick")
+
+/obj/item/toy/plush/monkey
+	name = "monkey plushie"
+	desc = "Its tag reads: 'Oop eek! I'm a chimpanzee!', with 'Now in JUMBO SIZE!' on the flipside."
```

Comment:
Maybe "The tag reads" instead of using pronouns here so people can call it whatever pronouns they want